.github/workflows/legacy-os.yml

```diff
@@ -16,3 +16,11 @@ jobs:
         run: |
           { { ! sudo scripts/lib/upgrade-zulip-stage-2 2>&1 >&3; } | tee upgrade.err; } 3>&1 >&2
           grep -Fq 'upgrade-zulip-stage-2: Unsupported platform: ubuntu 16.04' upgrade.err
+
+      - name: Report status
```

Comment:
This shouldn't have triggered the message https://chat.zulip.org/#narrow/stream/43-automated-testing/topic/ci_actions.20failing/near/1123124 since the test is not failing and repository is not zulip/zulip.